py-polars/pyproject.toml

```diff
@@ -56,6 +56,7 @@ plot = ["hvplot >= 0.9.1"]
 pyarrow = ["pyarrow >= 7.0.0"]
 pydantic = ["pydantic"]
 sqlalchemy = ["sqlalchemy", "pandas"]
+style = ["great-tables >= 0.8.0"]
```

Comment:
you'll need to add this to the `all` one, and maybe some other places - perhaps search "pydantic" or some other of these optional extra installs, and make sure the new one is added to all the right places?

Reply:
Alright -- should hopefully be added to the right places now? I...

* added the style dependency it to all
* added great_tables near pydantic in the mypy overrides